general/development/tools/nodejs.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: NodeJS and Grunt
+sidebar_position: 1
```

Comment:
Should this be 2? 

Do we always need to specify explicit order?

Reply:
Nope - it's intentionally 1 and therefore falls back to lexical sorting.

I've ordered these such that they're:
- during development (phpcs, and node)
- after development (mdk)
- automated actions on push (gha)
- automated actions during review (cibot)

Essentially trying to follow the logical order of when you'd expect to use or see them.